src/frontend/screens/Settings/sections/SystemInfo/gpu.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Paper from '@mui/material/Paper'
+import Typography from '@mui/material/Typography'
+import Grid from '@mui/material/Grid'
+
+import VendorLogo from './vendorLogo'
+
+import type { SystemInformation } from 'backend/utils/systeminfo'
+
+interface GPUCardProps {
+  gpu: SystemInformation['GPUs'][number]
+  gpuNumber: number
+  showNumber: boolean
+}
+
+function GPUCard({ gpu, gpuNumber, showNumber }: GPUCardProps) {
+  const {
+    vendorString,
+    deviceString,
+    deviceId,
+    vendorId,
+    subdeviceId,
+    subvendorId,
+    driverVersion
+  } = gpu
+  const { t } = useTranslation()
+
+  const headingText = showNumber
+    ? t('settings.systemInformation.gpuWithNumber', 'GPU {{no}}:', {
```

Comment:
```suggestion
    ? t('settings.systemInformation.gpuWithNumber', 'GPU {{number}}:', {
```

Just a suggestion, I think it gives a bit more context to translators